src/isPalindrome.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+/**
+ * Implement isPalindrome function:
+ *
+ * Given a string, determine if it is a palindrome, considering only
+ * alphanumeric characters and ignoring cases. Empty string is a valid
+ * palindrome.
```

Comment:
The description of what is `Palindrom` should be also here

Reply:
added palindrome description